pet-api/pet-api/Domain/Entities/Pagination/ProductParameters.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace pet_api.Domain.Entities.Pagination
+{
+    public class ProductParameters
+    {
+        const int maxPageSize = 50;
```

Comment:
use private const. const should be named from a big letter

Reply:
private modifier has been added